src/core/CoolBoard.cpp

```diff
@@ -637,3 +713,57 @@ void CoolBoard::mqttCallback(char *topic, byte *payload, unsigned int length) {
   temp[length] = '\0';
   this->update(temp);
 }
+
+void CoolBoard::otaUpdate() {
+  if (this->wifiManager.wifiMulti.run() == WL_CONNECTED && doOta) {
```

Comment:
no! DRY DRY DRY, Do not Repeat Yourself! You've already checked the `doOTA` flag before, you don't need to check it again!